### ISSUE TITLE
Tests fail in a brand new Rekit App

### ISSUE THREAD

**Issue by thehig** [supnate/rekit#56]
I've created a Rekit application using the latest toolkit. After creating the application I've tried to run the in-built tests, but I'm getting the following error (path redacted):

```
ERROR in ./src/features/home/RedditList.js
Module build failed: Error: C:/.../src/features/home/RedditList.js: We don't know what to do with this node type. We were previously a Statement but we can't fit in here?
```

There are many errors like this one, but they all have more-or-less the same structure

The key string here is *We don't know what to do with this node type. We were previously a Statement but we can't fit in here?* which appears to be coming from Babel

Some preliminary research suggests that this has something to do with the lines "export default class NAME extends SOMETHING", as splitting them to "class Name extends Something... " and "export default Name" seems to work.

**Comment by thehig** [comment]
Complete test run output (paths redacted to "...")

```
λ npm test

> giphyChallenge@0.0.1 test C:\...
> node ./tools/run_test.js

Running tests:  tests\**\*.test.js ...

ERROR in ./src/Root.js
Module build failed: Error: C:/.../src/Root.js: We don't know what to do with this node type. We were previously a Statement but we can't fit in here?
    at NodePath.insertBefore (C:\...\node_modules\babel-traverse\lib\path\modification.js:61:13)
    at VisitState.insertCounter (C:\...\node_modules\istanbul-lib-instrument\dist\visitor.js:197:22)
    at VisitState.insertStatementCounter (C:\...\node_modules\istanbul-lib-instrument\dist\visitor.js:224:18)
    at VisitState.coverStatement (C:\...\node_modules\istanbul-lib-instrument\dist\visitor.js:340:10)
    at C:\...\node_modules\istanbul-lib-instrument\dist\visitor.js:327:15
    at Array.forEach (native)
    at VisitState.wrappedEntry (C:\...\node_modules\istanbul-lib-instrument\dist\visitor.js:326:15)
    at NodePath._call (C:\...\node_modules\babel-traverse\lib\path\context.js:76:18)
    at NodePath.call (C:\...\node_modules\babel-traverse\lib\path\context.js:48:17)
    at NodePath.visit (C:\...\node_modules\babel-traverse\lib\path\context.js:105:12)
    at TraversalContext.visitQueue (C:\...\node_modules\babel-traverse\lib\context.js:150:16)
    at TraversalContext.visitSingle (C:\...\node_modules\babel-traverse\lib\context.js:108:19)
    at TraversalContext.visit (C:\...\node_modules\babel-traverse\lib\context.js:192:19)
    at Function.traverse.node (C:\...\node_modules\babel-traverse\lib\index.js:114:17)
    at NodePath.visit (C:\...\node_modules\babel-traverse\lib\path\context.js:115:19)
    at TraversalContext.visitQueue (C:\...\node_modules\babel-traverse\lib\context.js:150:16)
    at TraversalContext.visitMultiple (C:\...\node_modules\babel-traverse\lib\context.js:103:17)
    at TraversalContext.visit (C:\...\node_modules\babel-traverse\lib\context.js:190:19)
    at Function.traverse.node (C:\...\node_modules\babel-traverse\lib\index.js:114:17)
    at traverse (C:\...\node_modules\babel-traverse\lib\index.js:79:12)
    at NodePath.traverse (C:\...\node_modules\babel-traverse\lib\path\index.js:144:25)
    at Object.enter (C:\...\node_modules\istanbul-lib-instrument\dist\visitor.js:561:18)
    at PluginPass.enter (C:\...\node_modules\babel-plugin-istanbul\lib\index.js:79:23)
    at newFn (C:\...\node_modules\babel-traverse\lib\visitors.js:276:21)
    at NodePath._call (C:\...\node_modules\babel-traverse\lib\path\context.js:76:18)
    at NodePath.call (C:\...\node_modules\babel-traverse\lib\path\context.js:48:17)
    at NodePath.visit (C:\...\node_modules\babel-traverse\lib\path\context.js:105:12)
    at TraversalContext.visitQueue (C:\...\node_modules\babel-traverse\lib\context.js:150:16)
    at TraversalContext.visitSingle (C:\...\node_modules\babel-traverse\lib\context.js:108:19)
    at TraversalContext.visit (C:\...\node_modules\babel-traverse\lib\context.js:192:19)
    at Function.traverse.node (C:\...\node_modules\babel-traverse\lib\index.js:114:17)
    at traverse (C:\...\node_modules\babel-traverse\lib\index.js:79:12)
    at File.transform (C:\...\node_modules\babel-core\lib\transformation\file\index.js:548:35)
    at C:\...\node_modules\babel-core\lib\transformation\pipeline.js:50:19
    at File.wrap (C:\...\node_modules\babel-core\lib\transformation\file\index.js:564:16)
    at Pipeline.transform (C:\...\node_modules\babel-core\lib\transformation\pipeline.js:47:17)
    at transpile (C:\...\node_modules\babel-loader\lib\index.js:50:20)
    at Object.module.exports (C:\...\node_modules\babel-loader\lib\index.js:175:20)
 @ ./tests/Root.test.js 15:12-34
 @ ./tests object Object
 @ ./node_modules/mocha-webpack/lib/webpack/includeFilesLoader.js?{"include":["C://.../tests//before-all.js"]}!./.tmp/mocha-webpack/c982643df5c37431513b9e3ad1b96cbe-entry.js

ERROR in ./src/features/common/PageNotFound.js
Module build failed: Error: C:/.../src/features/common/PageNotFound.js: We don't know what to do with this node type. We were previously a Statement but we can't fit in here?
    at NodePath.insertBefore (C:\...\node_modules\babel-traverse\lib\path\modification.js:61:13)
    at VisitState.insertCounter (C:\...\node_modules\istanbul-lib-instrument\dist\visitor.js:197:22)
    at VisitState.insertStatementCounter (C:\...\node_modules\istanbul-lib-instrument\dist\visitor.js:224:18)
    at VisitState.coverStatement (C:\...\node_modules\istanbul-lib-instrument\dist\visitor.js:340:10)
    at C:\...\node_modules\istanbul-lib-instrument\dist\visitor.js:327:15
    at Array.forEach (native)
    at VisitState.wrappedEntry (C:\...\node_modules\istanbul-lib-instrument\dist\visitor.js:326:15)
    at NodePath._call (C:\...\node_modules\babel-traverse\lib\path\context.js:76:18)
    at NodePath.call (C:\...\node_modules\babel-traverse\lib\path\context.js:48:17)
    at NodePath.visit (C:\...\node_modules\babel-traverse\lib\path\context.js:105:12)
    at TraversalContext.visitQueue (C:\...\node_modules\babel-traverse\lib\context.js:150:16)
    at TraversalContext.visitSingle (C:\...\node_modules\babel-traverse\lib\context.js:108:19)
    at TraversalContext.visit (C:\...\node_modules\babel-traverse\lib\context.js:192:19)
    at Function.traverse.node (C:\...\node_modules\babel-traverse\lib\index.js:114:17)
    at NodePath.visit (C:\...\node_modules\babel-traverse\lib\path\context.js:115:19)
    at TraversalContext.visitQueue (C:\...\node_modules\babel-traverse\lib\context.js:150:16)
    at TraversalContext.visitMultiple (C:\...\node_modules\babel-traverse\lib\context.js:103:17)
    at TraversalContext.visit (C:\...\node_modules\babel-traverse\lib\context.js:190:19)
    at Function.traverse.node (C:\...\node_modules\babel-traverse\lib\index.js:114:17)
    at traverse (C:\...\node_modules\babel-traverse\lib\index.js:79:12)
    at NodePath.traverse (C:\...\node_modules\babel-traverse\lib\path\index.js:144:25)
    at Object.enter (C:\...\node_modules\istanbul-lib-instrument\dist\visitor.js:561:18)
    at PluginPass.enter (C:\...\node_modules\babel-plugin-istanbul\lib\index.js:79:23)
    at newFn (C:\...\node_modules\babel-traverse\lib\visitors.js:276:21)
    at NodePath._call (C:\...\node_modules\babel-traverse\lib\path\context.js:76:18)
    at NodePath.call (C:\...\node_modules\babel-traverse\lib\path\context.js:48:17)
    at NodePath.visit (C:\...\node_modules\babel-traverse\lib\path\context.js:105:12)
    at TraversalContext.visitQueue (C:\...\node_modules\babel-traverse\lib\context.js:150:16)
    at TraversalContext.visitSingle (C:\...\node_modules\babel-traverse\lib\context.js:108:19)
    at TraversalContext.visit (C:\...\node_modules\babel-traverse\lib\context.js:192:19)
    at Function.traverse.node (C:\...\node_modules\babel-traverse\lib\index.js:114:17)
    at traverse (C:\...\node_modules\babel-traverse\lib\index.js:79:12)
    at File.transform (C:\...\node_modules\babel-core\lib\transformation\file\index.js:548:35)
    at C:\...\node_modules\babel-core\lib\transformation\pipeline.js:50:19
    at File.wrap (C:\...\node_modules\babel-core\lib\transformation\file\index.js:564:16)
    at Pipeline.transform (C:\...\node_modules\babel-core\lib\transformation\pipeline.js:47:17)
    at transpile (C:\...\node_modules\babel-loader\lib\index.js:50:20)
    at Object.module.exports (C:\...\node_modules\babel-loader\lib\index.js:175:20)
 @ ./src/features/common/index.js 41:20-45
 @ ./tests/features/common/SimpleNav.test.js
 @ ./tests object Object
 @ ./node_modules/mocha-webpack/lib/webpack/includeFilesLoader.js?{"include":["C://.../tests//before-all.js"]}!./.tmp/mocha-webpack/c982643df5c37431513b9e3ad1b96cbe-entry.js

ERROR in ./src/features/common/SimpleNav.js
Module build failed: Error: C:/.../src/features/common/SimpleNav.js: We don't know what to do with this node type. We were previously a Statement but we can't fit in here?
    at NodePath.insertBefore (C:\...\node_modules\babel-traverse\lib\path\modification.js:61:13)
    at VisitState.insertCounter (C:\...\node_modules\istanbul-lib-instrument\dist\visitor.js:197:22)
    at VisitState.insertStatementCounter (C:\...\node_modules\istanbul-lib-instrument\dist\visitor.js:224:18)
    at VisitState.coverStatement (C:\...\node_modules\istanbul-lib-instrument\dist\visitor.js:340:10)
    at C:\...\node_modules\istanbul-lib-instrument\dist\visitor.js:327:15
    at Array.forEach (native)
    at VisitState.wrappedEntry (C:\...\node_modules\istanbul-lib-instrument\dist\visitor.js:326:15)
    at NodePath._call (C:\...\node_modules\babel-traverse\lib\path\context.js:76:18)
    at NodePath.call (C:\...\node_modules\babel-traverse\lib\path\context.js:48:17)
    at NodePath.visit (C:\...\node_modules\babel-traverse\lib\path\context.js:105:12)
    at TraversalContext.visitQueue (C:\...\node_modules\babel-traverse\lib\context.js:150:16)
    at TraversalContext.visitSingle (C:\...\node_modules\babel-traverse\lib\context.js:108:19)
    at TraversalContext.visit (C:\...\node_modules\babel-traverse\lib\context.js:192:19)
    at Function.traverse.node (C:\...\node_modules\babel-traverse\lib\index.js:114:17)
    at NodePath.visit (C:\...\node_modules\babel-traverse\lib\path\context.js:115:19)
    at TraversalContext.visitQueue (C:\...\node_modules\babel-traverse\lib\context.js:150:16)
    at TraversalContext.visitMultiple (C:\...\node_modules\babel-traverse\lib\context.js:103:17)
    at TraversalContext.visit (C:\...\node_modules\babel-traverse\lib\context.js:190:19)
    at Function.traverse.node (C:\...\node_modules\babel-traverse\lib\index.js:114:17)
    at traverse (C:\...\node_modules\babel-traverse\lib\index.js:79:12)
    at NodePath.traverse (C:\...\node_modules\babel-traverse\lib\path\index.js:144:25)
    at Object.enter (C:\...\node_modules\istanbul-lib-instrument\dist\visitor.js:561:18)
    at PluginPass.enter (C:\...\node_modules\babel-plugin-istanbul\lib\index.js:79:23)
    at newFn (C:\...\node_modules\babel-traverse\lib\visitors.js:276:21)
    at NodePath._call (C:\...\node_modules\babel-traverse\lib\path\context.js:76:18)
    at NodePath.call (C:\...\node_modules\babel-traverse\lib\path\context.js:48:17)
    at NodePath.visit (C:\...\node_modules\babel-traverse\lib\path\context.js:105:12)
    at TraversalContext.visitQueue (C:\...\node_modules\babel-traverse\lib\context.js:150:16)
    at TraversalContext.visitSingle (C:\...\node_modules\babel-traverse\lib\context.js:108:19)
    at TraversalContext.visit (C:\...\node_modules\babel-traverse\lib\context.js:192:19)
    at Function.traverse.node (C:\...\node_modules\babel-traverse\lib\index.js:114:17)
    at traverse (C:\...\node_modules\babel-traverse\lib\index.js:79:12)
    at File.transform (C:\...\node_modules\babel-core\lib\transformation\file\index.js:548:35)
    at C:\...\node_modules\babel-core\lib\transformation\pipeline.js:50:19
    at File.wrap (C:\...\node_modules\babel-core\lib\transformation\file\index.js:564:16)
    at Pipeline.transform (C:\...\node_modules\babel-core\lib\transformation\pipeline.js:47:17)
    at transpile (C:\...\node_modules\babel-loader\lib\index.js:50:20)
    at Object.module.exports (C:\...\node_modules\babel-loader\lib\index.js:175:20)
 @ ./src/features/common/index.js 32:17-39
 @ ./tests/features/common/SimpleNav.test.js
 @ ./tests object Object
 @ ./node_modules/mocha-webpack/lib/webpack/includeFilesLoader.js?{"include":["C://.../tests//before-all.js"]}!./.tmp/mocha-webpack/c982643df5c37431513b9e3ad1b96cbe-entry.js

ERROR in ./src/features/home/App.js
Module build failed: Error: C:/.../src/features/home/App.js: We don't know what to do with this node type. We were previously a Statement but we can't fit in here?
    at NodePath.insertBefore (C:\...\node_modules\babel-traverse\lib\path\modification.js:61:13)
    at VisitState.insertCounter (C:\...\node_modules\istanbul-lib-instrument\dist\visitor.js:197:22)
    at VisitState.insertStatementCounter (C:\...\node_modules\istanbul-lib-instrument\dist\visitor.js:224:18)
    at VisitState.coverStatement (C:\...\node_modules\istanbul-lib-instrument\dist\visitor.js:340:10)
    at C:\...\node_modules\istanbul-lib-instrument\dist\visitor.js:327:15
    at Array.forEach (native)
    at VisitState.wrappedEntry (C:\...\node_modules\istanbul-lib-instrument\dist\visitor.js:326:15)
    at NodePath._call (C:\...\node_modules\babel-traverse\lib\path\context.js:76:18)
    at NodePath.call (C:\...\node_modules\babel-traverse\lib\path\context.js:48:17)
    at NodePath.visit (C:\...\node_modules\babel-traverse\lib\path\context.js:105:12)
    at TraversalContext.visitQueue (C:\...\node_modules\babel-traverse\lib\context.js:150:16)
    at TraversalContext.visitSingle (C:\...\node_modules\babel-traverse\lib\context.js:108:19)
    at TraversalContext.visit (C:\...\node_modules\babel-traverse\lib\context.js:192:19)
    at Function.traverse.node (C:\...\node_modules\babel-traverse\lib\index.js:114:17)
    at NodePath.visit (C:\...\node_modules\babel-traverse\lib\path\context.js:115:19)
    at TraversalContext.visitQueue (C:\...\node_modules\babel-traverse\lib\context.js:150:16)
    at TraversalContext.visitMultiple (C:\...\node_modules\babel-traverse\lib\context.js:103:17)
    at TraversalContext.visit (C:\...\node_modules\babel-traverse\lib\context.js:190:19)
    at Function.traverse.node (C:\...\node_modules\babel-traverse\lib\index.js:114:17)
    at traverse (C:\...\node_modules\babel-traverse\lib\index.js:79:12)
    at NodePath.traverse (C:\...\node_modules\babel-traverse\lib\path\index.js:144:25)
    at Object.enter (C:\...\node_modules\istanbul-lib-instrument\dist\visitor.js:561:18)
    at PluginPass.enter (C:\...\node_modules\babel-plugin-istanbul\lib\index.js:79:23)
    at newFn (C:\...\node_modules\babel-traverse\lib\visitors.js:276:21)
    at NodePath._call (C:\...\node_modules\babel-traverse\lib\path\context.js:76:18)
    at NodePath.call (C:\...\node_modules\babel-traverse\lib\path\context.js:48:17)
    at NodePath.visit (C:\...\node_modules\babel-traverse\lib\path\context.js:105:12)
    at TraversalContext.visitQueue (C:\...\node_modules\babel-traverse\lib\context.js:150:16)
    at TraversalContext.visitSingle (C:\...\node_modules\babel-traverse\lib\context.js:108:19)
    at TraversalContext.visit (C:\...\node_modules\babel-traverse\lib\context.js:192:19)
    at Function.traverse.node (C:\...\node_modules\babel-traverse\lib\index.js:114:17)
    at traverse (C:\...\node_modules\babel-traverse\lib\index.js:79:12)
    at File.transform (C:\...\node_modules\babel-core\lib\transformation\file\index.js:548:35)
    at C:\...\node_modules\babel-core\lib\transformation\pipeline.js:50:19
    at File.wrap (C:\...\node_modules\babel-core\lib\transformation\file\index.js:564:16)
    at Pipeline.transform (C:\...\node_modules\babel-core\lib\transformation\pipeline.js:47:17)
    at transpile (C:\...\node_modules\babel-loader\lib\index.js:50:20)
    at Object.module.exports (C:\...\node_modules\babel-loader\lib\index.js:175:20)
 @ ./tests/features/home/App.test.js 11:11-52
 @ ./tests object Object
 @ ./node_modules/mocha-webpack/lib/webpack/includeFilesLoader.js?{"include":["C://.../tests//before-all.js"]}!./.tmp/mocha-webpack/c982643df5c37431513b9e3ad1b96cbe-entry.js

ERROR in ./src/features/home/DefaultPage.js
Module build failed: Error: C:/.../src/features/home/DefaultPage.js: We don't know what to do with this node type. We were previously a Statement but we can't fit in here?
    at NodePath.insertBefore (C:\...\node_modules\babel-traverse\lib\path\modification.js:61:13)
    at VisitState.insertCounter (C:\...\node_modules\istanbul-lib-instrument\dist\visitor.js:197:22)
    at VisitState.insertStatementCounter (C:\...\node_modules\istanbul-lib-instrument\dist\visitor.js:224:18)
    at VisitState.coverStatement (C:\...\node_modules\istanbul-lib-instrument\dist\visitor.js:340:10)
    at C:\...\node_modules\istanbul-lib-instrument\dist\visitor.js:327:15
    at Array.forEach (native)
    at VisitState.wrappedEntry (C:\...\node_modules\istanbul-lib-instrument\dist\visitor.js:326:15)
    at NodePath._call (C:\...\node_modules\babel-traverse\lib\path\context.js:76:18)
    at NodePath.call (C:\...\node_modules\babel-traverse\lib\path\context.js:48:17)
    at NodePath.visit (C:\...\node_modules\babel-traverse\lib\path\context.js:105:12)
    at TraversalContext.visitQueue (C:\...\node_modules\babel-traverse\lib\context.js:150:16)
    at TraversalContext.visitSingle (C:\...\node_modules\babel-traverse\lib\context.js:108:19)
    at TraversalContext.visit (C:\...\node_modules\babel-traverse\lib\context.js:192:19)
    at Function.traverse.node (C:\...\node_modules\babel-traverse\lib\index.js:114:17)
    at NodePath.visit (C:\...\node_modules\babel-traverse\lib\path\context.js:115:19)
    at TraversalContext.visitQueue (C:\...\node_modules\babel-traverse\lib\context.js:150:16)
    at TraversalContext.visitMultiple (C:\...\node_modules\babel-traverse\lib\context.js:103:17)
    at TraversalContext.visit (C:\...\node_modules\babel-traverse\lib\context.js:190:19)
    at Function.traverse.node (C:\...\node_modules\babel-traverse\lib\index.js:114:17)
    at traverse (C:\...\node_modules\babel-traverse\lib\index.js:79:12)
    at NodePath.traverse (C:\...\node_modules\babel-traverse\lib\path\index.js:144:25)
    at Object.enter (C:\...\node_modules\istanbul-lib-instrument\dist\visitor.js:561:18)
    at PluginPass.enter (C:\...\node_modules\babel-plugin-istanbul\lib\index.js:79:23)
    at newFn (C:\...\node_modules\babel-traverse\lib\visitors.js:276:21)
    at NodePath._call (C:\...\node_modules\babel-traverse\lib\path\context.js:76:18)
    at NodePath.call (C:\...\node_modules\babel-traverse\lib\path\context.js:48:17)
    at NodePath.visit (C:\...\node_modules\babel-traverse\lib\path\context.js:105:12)
    at TraversalContext.visitQueue (C:\...\node_modules\babel-traverse\lib\context.js:150:16)
    at TraversalContext.visitSingle (C:\...\node_modules\babel-traverse\lib\context.js:108:19)
    at TraversalContext.visit (C:\...\node_modules\babel-traverse\lib\context.js:192:19)
    at Function.traverse.node (C:\...\node_modules\babel-traverse\lib\index.js:114:17)
    at traverse (C:\...\node_modules\babel-traverse\lib\index.js:79:12)
    at File.transform (C:\...\node_modules\babel-core\lib\transformation\file\index.js:548:35)
    at C:\...\node_modules\babel-core\lib\transformation\pipeline.js:50:19
    at File.wrap (C:\...\node_modules\babel-core\lib\transformation\file\index.js:564:16)
    at Pipeline.transform (C:\...\node_modules\babel-core\lib\transformation\pipeline.js:47:17)
    at transpile (C:\...\node_modules\babel-loader\lib\index.js:50:20)
    at Object.module.exports (C:\...\node_modules\babel-loader\lib\index.js:175:20)
 @ ./tests/features/home/DefaultPage.test.js 19:19-68
 @ ./tests object Object
 @ ./node_modules/mocha-webpack/lib/webpack/includeFilesLoader.js?{"include":["C://.../tests//before-all.js"]}!./.tmp/mocha-webpack/c982643df5c37431513b9e3ad1b96cbe-entry.js

ERROR in ./src/features/home/RedditList.js
Module build failed: Error: C:/.../src/features/home/RedditList.js: We don't know what to do with this node type. We were previously a Statement but we can't fit in here?
    at NodePath.insertBefore (C:\...\node_modules\babel-traverse\lib\path\modification.js:61:13)
    at VisitState.insertCounter (C:\...\node_modules\istanbul-lib-instrument\dist\visitor.js:197:22)
    at VisitState.insertStatementCounter (C:\...\node_modules\istanbul-lib-instrument\dist\visitor.js:224:18)
    at VisitState.coverStatement (C:\...\node_modules\istanbul-lib-instrument\dist\visitor.js:340:10)
    at C:\...\node_modules\istanbul-lib-instrument\dist\visitor.js:327:15
    at Array.forEach (native)
    at VisitState.wrappedEntry (C:\...\node_modules\istanbul-lib-instrument\dist\visitor.js:326:15)
    at NodePath._call (C:\...\node_modules\babel-traverse\lib\path\context.js:76:18)
    at NodePath.call (C:\...\node_modules\babel-traverse\lib\path\context.js:48:17)
    at NodePath.visit (C:\...\node_modules\babel-traverse\lib\path\context.js:105:12)
    at TraversalContext.visitQueue (C:\...\node_modules\babel-traverse\lib\context.js:150:16)
    at TraversalContext.visitSingle (C:\...\node_modules\babel-traverse\lib\context.js:108:19)
    at TraversalContext.visit (C:\...\node_modules\babel-traverse\lib\context.js:192:19)
    at Function.traverse.node (C:\...\node_modules\babel-traverse\lib\index.js:114:17)
    at NodePath.visit (C:\...\node_modules\babel-traverse\lib\path\context.js:115:19)
    at TraversalContext.visitQueue (C:\...\node_modules\babel-traverse\lib\context.js:150:16)
    at TraversalContext.visitMultiple (C:\...\node_modules\babel-traverse\lib\context.js:103:17)
    at TraversalContext.visit (C:\...\node_modules\babel-traverse\lib\context.js:190:19)
    at Function.traverse.node (C:\...\node_modules\babel-traverse\lib\index.js:114:17)
    at traverse (C:\...\node_modules\babel-traverse\lib\index.js:79:12)
    at NodePath.traverse (C:\...\node_modules\babel-traverse\lib\path\index.js:144:25)
    at Object.enter (C:\...\node_modules\istanbul-lib-instrument\dist\visitor.js:561:18)
    at PluginPass.enter (C:\...\node_modules\babel-plugin-istanbul\lib\index.js:79:23)
    at newFn (C:\...\node_modules\babel-traverse\lib\visitors.js:276:21)
    at NodePath._call (C:\...\node_modules\babel-traverse\lib\path\context.js:76:18)
    at NodePath.call (C:\...\node_modules\babel-traverse\lib\path\context.js:48:17)
    at NodePath.visit (C:\...\node_modules\babel-traverse\lib\path\context.js:105:12)
    at TraversalContext.visitQueue (C:\...\node_modules\babel-traverse\lib\context.js:150:16)
    at TraversalContext.visitSingle (C:\...\node_modules\babel-traverse\lib\context.js:108:19)
    at TraversalContext.visit (C:\...\node_modules\babel-traverse\lib\context.js:192:19)
    at Function.traverse.node (C:\...\node_modules\babel-traverse\lib\index.js:114:17)
    at traverse (C:\...\node_modules\babel-traverse\lib\index.js:79:12)
    at File.transform (C:\...\node_modules\babel-core\lib\transformation\file\index.js:548:35)
    at C:\...\node_modules\babel-core\lib\transformation\pipeline.js:50:19
    at File.wrap (C:\...\node_modules\babel-core\lib\transformation\file\index.js:564:16)
    at Pipeline.transform (C:\...\node_modules\babel-core\lib\transformation\pipeline.js:47:17)
    at transpile (C:\...\node_modules\babel-loader\lib\index.js:50:20)
    at Object.module.exports (C:\...\node_modules\babel-loader\lib\index.js:175:20)
 @ ./src/features/home/index.js 41:18-41
 @ ./tests/features/home/RedditList.test.js
 @ ./tests object Object
 @ ./node_modules/mocha-webpack/lib/webpack/includeFilesLoader.js?{"include":["C://.../tests//before-all.js"]}!./.tmp/mocha-webpack/c982643df5c37431513b9e3ad1b96cbe-entry.js

ERROR in ./src/features/home/TestPage.js
Module build failed: Error: C:/.../src/features/home/TestPage.js: We don't know what to do with this node type. We were previously a Statement but we can't fit in here?
    at NodePath.insertBefore (C:\...\node_modules\babel-traverse\lib\path\modification.js:61:13)
    at VisitState.insertCounter (C:\...\node_modules\istanbul-lib-instrument\dist\visitor.js:197:22)
    at VisitState.insertStatementCounter (C:\...\node_modules\istanbul-lib-instrument\dist\visitor.js:224:18)
    at VisitState.coverStatement (C:\...\node_modules\istanbul-lib-instrument\dist\visitor.js:340:10)
    at C:\...\node_modules\istanbul-lib-instrument\dist\visitor.js:327:15
    at Array.forEach (native)
    at VisitState.wrappedEntry (C:\...\node_modules\istanbul-lib-instrument\dist\visitor.js:326:15)
    at NodePath._call (C:\...\node_modules\babel-traverse\lib\path\context.js:76:18)
    at NodePath.call (C:\...\node_modules\babel-traverse\lib\path\context.js:48:17)
    at NodePath.visit (C:\...\node_modules\babel-traverse\lib\path\context.js:105:12)
    at TraversalContext.visitQueue (C:\...\node_modules\babel-traverse\lib\context.js:150:16)
    at TraversalContext.visitSingle (C:\...\node_modules\babel-traverse\lib\context.js:108:19)
    at TraversalContext.visit (C:\...\node_modules\babel-traverse\lib\context.js:192:19)
    at Function.traverse.node (C:\...\node_modules\babel-traverse\lib\index.js:114:17)
    at NodePath.visit (C:\...\node_modules\babel-traverse\lib\path\context.js:115:19)
    at TraversalContext.visitQueue (C:\...\node_modules\babel-traverse\lib\context.js:150:16)
    at TraversalContext.visitMultiple (C:\...\node_modules\babel-traverse\lib\context.js:103:17)
    at TraversalContext.visit (C:\...\node_modules\babel-traverse\lib\context.js:190:19)
    at Function.traverse.node (C:\...\node_modules\babel-traverse\lib\index.js:114:17)
    at traverse (C:\...\node_modules\babel-traverse\lib\index.js:79:12)
    at NodePath.traverse (C:\...\node_modules\babel-traverse\lib\path\index.js:144:25)
    at Object.enter (C:\...\node_modules\istanbul-lib-instrument\dist\visitor.js:561:18)
    at PluginPass.enter (C:\...\node_modules\babel-plugin-istanbul\lib\index.js:79:23)
    at newFn (C:\...\node_modules\babel-traverse\lib\visitors.js:276:21)
    at NodePath._call (C:\...\node_modules\babel-traverse\lib\path\context.js:76:18)
    at NodePath.call (C:\...\node_modules\babel-traverse\lib\path\context.js:48:17)
    at NodePath.visit (C:\...\node_modules\babel-traverse\lib\path\context.js:105:12)
    at TraversalContext.visitQueue (C:\...\node_modules\babel-traverse\lib\context.js:150:16)
    at TraversalContext.visitSingle (C:\...\node_modules\babel-traverse\lib\context.js:108:19)
    at TraversalContext.visit (C:\...\node_modules\babel-traverse\lib\context.js:192:19)
    at Function.traverse.node (C:\...\node_modules\babel-traverse\lib\index.js:114:17)
    at traverse (C:\...\node_modules\babel-traverse\lib\index.js:79:12)
    at File.transform (C:\...\node_modules\babel-core\lib\transformation\file\index.js:548:35)
    at C:\...\node_modules\babel-core\lib\transformation\pipeline.js:50:19
    at File.wrap (C:\...\node_modules\babel-core\lib\transformation\file\index.js:564:16)
    at Pipeline.transform (C:\...\node_modules\babel-core\lib\transformation\pipeline.js:47:17)
    at transpile (C:\...\node_modules\babel-loader\lib\index.js:50:20)
    at Object.module.exports (C:\...\node_modules\babel-loader\lib\index.js:175:20)
 @ ./tests/features/home/TestPage.test.js 11:16-62
 @ ./tests object Object
 @ ./node_modules/mocha-webpack/lib/webpack/includeFilesLoader.js?{"include":["C://.../tests//before-all.js"]}!./.tmp/mocha-webpack/c982643df5c37431513b9e3ad1b96cbe-entry.js
Warning: ReactTestUtils has been moved to react-dom/test-utils. Update references to remove this warning.

=============================== Coverage summary ===============================
Statements   : 57.41% ( 31/54 )
Branches     : 52.94% ( 9/17 )
Functions    : 35.29% ( 6/17 )
Lines        : 58.49% ( 31/53 )
================================================================================
child_process.js:631
    throw err;
    ^

Error: Command failed: nyc --report-dir=coverage mocha-webpack --colors --include tests/before-all.js --webpack-config webpack.test.config.js "C:\...\tests\**\*.test.js"
    at checkExecSyncError (child_process.js:588:13)
    at execSync (child_process.js:628:13)
    at Function.npmExecSync (C:\...\node_modules\npm-run\index.js:53:10)
    at Object.<anonymous> (C:\...\tools\run_test.js:51:8)
    at Module._compile (module.js:569:30)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
    at Function.Module._load (module.js:458:3)
    at Function.Module.runMain (module.js:605:10)
```

**Comment by supnate** [comment]
Fixed in rekit@2.1.0.
